lib/App.js

```diff
@@ -1369,9 +1443,10 @@ class App {
     await fse.ensureDir( path.join(driverPath, 'assets') );
     await fse.ensureDir( path.join(driverPath, 'assets', 'images') );
 
+    const fileExtension = this.usesTypeScript() ? '.ts' : '.js';
```

Comment:
lets change the `templatePath` instead of the file extensions. so like `/assets/templates/typescript/app/drivers` or `/assets/templates/javascript/app/drivers`